### ISSUE TITLE
doc typo: an extra "]"

### ISSUE THREAD

**Issue by muzimuzhi** [T-F-S/tcolorbox#127]
Position
https://github.com/T-F-S/tcolorbox/blob/cc675797483e436768e59cc21e3d9fc4d1823616/doc/latex/tcolorbox/tcolorbox.doc.hooks.tex#L190

Proposal
```diff
diff --git a/doc/latex/tcolorbox/tcolorbox.doc.hooks.tex b/doc/latex/tcolorbox/tcolorbox.doc.hooks.tex
index f3fab22..da47d26 100644
--- a/doc/latex/tcolorbox/tcolorbox.doc.hooks.tex
+++ b/doc/latex/tcolorbox/tcolorbox.doc.hooks.tex
@@ -187,7 +187,7 @@ from page \pageref{subsec:overlays}.
     \begin{scope}[shift={([xshift=\x]frame.north west)}]
       \path[draw=green!65!black,fill=green!10,line width=1mm] (0,0) arc (0:180:5mm);
       \path[fill=black] (-0.2,0) arc (0:180:1mm);
-    \end{scope}}}]}}
+    \end{scope}}}}}
 \tcbset{ribbon/.style={overlay app={%
   \path[fill=blue!75!white,draw=blue,double=white!85!blue,
     preaction={opacity=0.6,fill=blue!75!white},
```

The extra `]` is finally used in `pgfpicture` environment and will (silently) write message `Missing character: There is no ] in font nullfont!` to log file. I've checked that there is only one such message in the log file after a local doc compilation.

**Comment by T-F-S** [comment]
Happy new year - and thank you for reporting this.

**Comment by muzimuzhi** [comment]
Happy new year! Guess what I found: [How should I answer 'Happy New Year'?](https://ell.stackexchange.com/questions/15132/how-should-i-answer-happy-new-year/15140)

**Comment by T-F-S** [comment]
Fixed with version 4.50 (2021/05/21)

**Comment by muzimuzhi** [comment]
Thanks. Just for the record, fixed by [this line](https://github.com/T-F-S/tcolorbox/commit/00f2d150b7152a7765fc3a4ba7c19b8368cc7283#diff-ab8ae6633941b67c3362d09a9174c336597e4df88b1f66d748d641d2379ccebd).
